clogMonitor/src/components/TimeRange.js

```diff
@@ -1,37 +1,149 @@
 import React from 'react';
 import { FormControl, FormHelperText, InputLabel, OutlinedInput, Stack } from '@mui/material';
+import { MenuItem, Select } from '@mui/material';
+import moment from 'moment';
+
+/**
+ * Returns true if DST is in effect in user's timezone 
+ * 
+ * @param {Object} props
+ * @param {string} props.time - The string representing the current time, format: YYYY-MM-DDTHH:mm:ss
```

Comment:
For this and the other exported functions, there is not a props object being passed, it's just regular arguments
For example, this one should be:
```
@param {string} time - The string representing the current time, format: YYYY-MM-DDTHH:mm:ss
```